docs/medical-api/api-reference/patient/delete-patient.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: "Delete Patient"
+description: "Removes the specified Patient."
+api: "DELETE /medical/v1/patient/{id}"
+---
+
+## Query Params
+
+<ParamField query="facilityId" type="string" required>
+  The ID of the Facility where the patient is receiving care.
+</ParamField>
+
+## Path Params
+
+<ParamField path="id" type="string" required>
+  The ID of the Patient to delete.
+</ParamField>
+
+## Response
+
+204 - no content, upon successful delete.
```

Comment:
nit: the 204 with backticks, ie `204` would make the response code more obvious at a glance